### PR TITLE
[Feature]add edsr result && super-resolution ncnn-int8 config

### PR DESCRIPTION
**Pull request by OldDreamInWind**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

1. use mmdeploy to deploy edsr model in mmediting to ncnn-int8

## Modification

1. add edsr ncnn-int8 result in quantization.md
2. add super-resolution ncnn-int8 config
3. compare task type when get input_data in tools/onnx2ncnn_quant_table.py

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
4. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
5. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
6. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by OldDreamInWind** [comment]
First of all, modify dataset path in mmediting/configs/restorers/edsr/edsr_x2c64b16_g1_300k_div2k.py

command to run：
1. export MODEL_PATH=/path/to/mmediting/configs/restorers/edsr/edsr_x2c64b16_g1_300k_div2k.py
2. export MODEL_CONFIG=https://download.openmmlab.com/mmediting/restorers/edsr/edsr_x2c64b16_1x16_300k_div2k_20200604-19fe95ea.pth
3. python3 tools/deploy.py configs/mmedit/super-resolution/super-resolution_ncnn-int8_dynamic.py ${MODEL_CONFIG} ${MODEL_PATH} ../screenshot.png --work-dir ../mmdeploy-mmediting/ --device cpu --quant
4. python tools/test.py configs/mmedit/super-resolution/super-resolution_ncnn-int8_dynamic.py ${MODEL_CONFIG} --model ~/workspace/mmdeploy-mmediting/end2end_int8.param ~/workspace/mmdeploy-mmediting/end2end_int8.bin --device cpu

**Comment by OldDreamInWind** [comment]
[知乎链接:https://zhuanlan.zhihu.com/p/568703127](https://zhuanlan.zhihu.com/p/568703127)

**Comment by RunningLeon** [comment]
@OldDreamInWind Could fix lint error with pre-commit

**Comment by tpoisonooo** [comment]
Also update  https://github.com/open-mmlab/mmdeploy/blob/master/docs/zh_cn/03-benchmark/benchmark.md.

**Comment by tpoisonooo** [comment]
How to fix lint:
```bash
$ python3 -m pip install pre-commit
$ cd path/to/mmdeploy
$ pre-commit install
$ pre-commit run --all-files
```

**Review by tpoisonooo — APPROVED** [review]
LGTM after fix CI lint

**Comment by OldDreamInWind** [comment]
done.

File: docs/en/03-benchmark/benchmark.md [hunk 1]
```diff
@@ -1061,6 +1090,35 @@ Users can directly test the performance through [how_to_evaluate_a_model.md](../
     <td align="center">0.8497</td>
     <td align="center">0.8469</td>
     <td align="center"> - </td>
+    <td align="center">-</td>
+    <td align="center">-</td>
+  </tr>
+    <tr>
+    <td align="center" rowspan="2"><a href="https://github.com/open-mmlab/mmediting/blob/master/configs/restorers/edsr/edsr_x2c64b16_g1_300k_div2k.py">EDSRx2</a></td>
```
Comment (RunningLeon) [inline docs/en/03-benchmark/benchmark.md:1097]: Why not test `EDSRx4` with ncnn-int8?

File: docs/en/03-benchmark/benchmark.md [hunk 2]
```diff
@@ -1061,6 +1090,35 @@ Users can directly test the performance through [how_to_evaluate_a_model.md](../
...
+    <td align="center" rowspan="2">Super Resolution</td>
+    <td align="center" rowspan="2">Set5</td>
+    <td align="center">PSNR</td>
+    <td align="center">-</td>
```
Comment (RunningLeon) [inline docs/en/03-benchmark/benchmark.md:1101]: Could you fill the metric values of PyTorch column?

Comment (OldDreamInWind) [reply docs/en/03-benchmark/benchmark.md]: sure, I filled the metric values of PyTorch column just now

File: docs/en/03-benchmark/benchmark.md (hunk 1 above)
Comment (OldDreamInWind) [reply docs/en/03-benchmark/benchmark.md:1097]: sorry, I found edsr is x4 in benchmark.md after testing EDSRx2 with ncnn-int8. Converting model is a little slow. I will test EDSRx4 with ncnn-int8 in next week.

**Comment by OldDreamInWind** [comment]
when I try to convert EDSRx4, it seems to have encountered some problems. I'll try to solve it.
```
mmdeploy - ERROR - `mmdeploy.apis.pytorch2onnx.torch2onnx` with Call id: 0 failed. exit.
```

File: docs/en/03-benchmark/benchmark.md (hunk 1 above)
Comment (OldDreamInWind) [reply docs/en/03-benchmark/benchmark.md:1097]: have tested EDSRx4 and updated the result in this pr

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**